streaming/src/main/scala/org/apache/gearpump/streaming/DAG.scala

```diff
@@ -51,6 +52,16 @@ object DAG {
       taskDescription.equals(node)
     }.get._1
   }
+
+  implicit val writer = upickle.Writer[DAG] {
```

Comment:
I should add that no explicit writer is needed for AppDescription since the other field (name) is handled implicitly. In the case of DAG we are sort of rolling our own JSON. We may want to have an implicit conversion (other than writer[T]) which is more JSON friendly. Will look at this more closely when the DAG widget evolves.